package.json

```diff
@@ -19,7 +19,7 @@
     "url": "https://github.com/prettier/prettier-vscode/issues"
   },
   "engines": {
-    "vscode": "^1.5.0"
+    "vscode": "^1.15.0"
```

Comment:
I had to do this to update `vscode.d.ts` with the multi-root stuff. Is this OK?

I've maintained backwards compatibility at runtime, all I really need is the definitions.